### PR TITLE
docs(readme.md): add some examples of do and don't

### PR DESCRIPTION
**Pull request by akoshelev**
Adds some illustrative examples of how to write code according to the conventions proposed. 
Provides code examples for `this` usage, specifying visibility for class members and using var in favor of explicit type.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ public class Example
 ```C#
 if (a == "b") 
 {
-    Consloe.WriteLine("A equals to B");
+    Console.WriteLine("A equals to B");
```
Comment (maxbrodin) [inline README.md:55]: Nice catch :eyes:

File: README.md [hunk 2]
```diff
@@ -100,7 +100,53 @@ public String TrimString(String s) {
 ## Class members
 
 - Avoid `this.` unless absolutely necessary.
-- Always specify member visiblity, even if it's the default (i.e. `private string foo;` not `string _foo;`).
+
+DO
```
Comment (maxbrodin) [inline README.md]: Minor: Could we use `&#10003; **DO**` and `&#10007; **DO NOT**` markup for &#10003; **DO** and &#10007; **DO NOT**

File: README.md [hunk 3]
```diff
@@ -100,7 +100,53 @@ public String TrimString(String s) {
...
+```
+public class Node
```
Comment (maxbrodin) [inline README.md]: Minor: I thought about

```
public UserController(IUserSettingsRepository userSettingsRepository)
{
    _userSettingsRepository = userSettingsRepository;
}
```

and

```
public UserController(IUserSettingsRepository userSettingsRepository)
{
    this._userSettingsRepository = userSettingsRepository;
}
```

to make it simplier and more related to our code :wink: But you sample works for me too :+1:

**Comment by maxbrodin** [comment]
Looks good :+1: 
Please, address `DO` and `DO NOT` markup and merge it in.

File: README.md (hunk 2 above)
Comment (akoshelev) [inline README.md]: I like the idea. Will do

File: README.md (hunk 3 above)
Comment (akoshelev) [inline README.md]: Both are the clear examples of how to use `this`. Let's use your code to demonstrate the approach - I don't mind.

**Comment by akoshelev** [comment]
All done. As you're the owner of this repository I'd recommend that you'll merge this PR to master. It makes the process more robust and adopted to the github practices.

File: README.md [hunk 4]
```diff
@@ -100,7 +100,37 @@ public String TrimString(String s) {
...
+- Always specify member visiblity, even if it's the default
+
+&#10003; **DO**
+```
+private string _foo;
+```
+
+&#10007; **DO NOT**
+```
+string _foo
```
Comment (maxbrodin) [inline README.md]: `;` missed should be `string _foo;`

**Comment by akoshelev** [comment]
done

**Pull request merged**